test/mode/normalModeTests/motions.test.ts

```diff
@@ -639,4 +639,20 @@ suite('Motions in Normal Mode', () => {
     keysPressed: '<right>',
     end: ['blah', 'duh', 'd|ur', 'hur'],
   });
+
+  newTest({
+    title: 'Can handle <C-u> key',
+    start: ['blah', 'duh', 'dur', 'hu|r'],
+    keysPressed: '<C-u>',
+    end: ['blah', 'duh', 'du|r', 'hur'],
+  });
+
+  /*
+  newTest({
+    title: 'Can handle <C-d> key',
+    start: ['bla|h', 'duh', 'dur', 'hur'],
+    keysPressed: '<C-d>',
+    end: ['blah', 'duh', 'dur', 'hu|r'],
```

Comment:
Why is this commented out?

Reply:
The test should work but I think the scrolling makes it incorrect

Reply:
I have tried to solve this but cannot make the test pass, so I will leave it commented

Reply:
Please fix this test.